ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/gui/BoardPanel.java

```diff
@@ -50,17 +52,33 @@ public BoardPanel(BoardSize boardSize, int boardIndex, GlassPane globalGlassPane
         };
 
         setOpaque(false);
-        setLayout(new GridLayout(boardSize.width + 1, boardSize.height));
-        setPreferredSize(new Dimension((boardSize.width + 1) * 48, (boardSize.height + 1) * 48));
-        createGrid(boardSize);
+        setLayout(new GridLayout(mBoardSize.width + 1, mBoardSize.height));
```

Comment:
Also, if we only ever use `mBoardSize.width + 1` and `mBoardSize.height + 1`, how about we initialize `mBoardSize` to that by creating a new `BoardSize` from the one passed in?